### DISCUSSION TITLE
Multiple requests from apple.com on my adguard-home installed on a server.

### DISCUSSION THREAD

**Discussion by ghost** [AdguardTeam/AdGuardHome#6271] [General]
I had multiple requests from apple.com and there's no apple device in my lan. What can it be?
My guess is antennapod, the podcast app. As client I see 194.3.0.0 which I don't understand why. How to set AGH in a way to see my LAN clients, like 192.168.1.xxx?

**Comment by o1x3** [comment]
Is your server publicly exposed?

**Reply by ghost** [reply]
If you mean having port 53 open then yes.

**Comment by ghost** [comment]
It got a lot worse today:
thousands of requests to apple.com:
![immagine](https://github.com/AdguardTeam/AdGuardHome/assets/146669079/8d80b950-5b9d-4208-bca1-6e9714bed5a8)

**Reply by o1x3** [reply]
I had the same issue just yesterday. Closing port 53 incoming traffic fixed it. I only use DoH/DoT so it wasn't an issue for me

**Comment by ppfeufer** [comment]
You seem to have a public instance, so stuff like that is bound to happen and expected at some point.

**Reply by ghost** [reply]
I do, is it a security problem to run adh like that?

**Reply by ppfeufer** [reply]
Well, let's say, as I did that, my server hoster was contacted by the [BSI (German Federal Office for Information Security)](https://www.bsi.bund.de/EN/Home/home_node.html) via mail which they patched through to me, so I had a little chat with them. Bottom line, they told me that I sure can run it like that, but it's not the best idea to do so and if something fishy (DNS poisoning and so on) happens through my public AGH instance I would be liable for it. So I moved it behind a VPN, as it's for my mobile devices only anyway, and I'm running a VPN on them now for this.

The AGH instance for my PC and anything else in my household that needs an internet connection is running on a Raspberry Pi in my network at home, and is not exposed publically at all.

So yeah, a public AGH instance is probably not the best idea, as it has the potential to temper with DNS resolution, as you can add your own DNS rewrites, which would be public as well. And I guess that is the part the BSI took an issue with.

**Reply by ghost** [reply]
Port closed, thanks for the detailed answer.
